interface-elements-for-desktop/articles/pdf-viewer.md

```diff
@@ -4,13 +4,15 @@ author: Alexander Temnov
 legacyId: 17451
 ---
 # PDF Viewer
-This section describes the capabilities of the **PDF Viewer**.
 
-The PDF Viewer used to displaying PDF files and supports various document content such as text, images, vector graphics, etc.
+This section describes the **PDF Viewer** capabilities.
```

Comment:
"the **PDF Viewer** capabilities" There aren't PDF Viewer capabilities, there are the capabilities of the PDF Viewer or the PDF Viewer's capabilities. However, this does not explain what you mean when you say **capabilities**.